tasks/main.yml

```diff
@@ -7,15 +7,20 @@
   sudo: yes
   apt: pkg=apache2 state=latest
 
-- name: Install Apache Modules
-  apache2_module: state=present name={{ item }}
-  notify: restart apache
-  with_items:
-    - rewrite
-    - vhost_alias
-    - headers
-    - expires
-    - filter
+- name: Install Apache Modules rewrite
```

Comment:
@debo the latest, 2.0.0.2 and the same at TravisCI [#30](https://travis-ci.org/phansible/role-apache/builds/104153161). As an option, we can use a shell command (doing the same that the python [module](https://github.com/ansible/ansible-modules-core/blob/devel/web_infrastructure/apache2_module.py)) and chnge the playbook command to run with -vvvv (verbose):

```yml
- name: Enable Apache modules
  shell:
    a2enmod {{ item }}
  notify: restart apache
  with_items:
    - rewrite
    - vhost_alias
    - headers
    - expires
    - filter
```